### PR TITLE
Update Korean Translation to 5275244

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

File: docs/docs/10.7-update.ko-KR.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ next: pure-render-mixin-ko-KR.html
 
 React에서는 mutation을 포함해 어떤 데이터 관리 방식도 사용하실 수 있습니다. 하지만 애플리케이션의 성능이 중요한 부분에서 불변의(immutable) 데이터를 사용할 수 있다면, 쉽게 빠른 `shouldComponentUpdate()` 메소드를 구현해 애플리케이션의 속도를 크게 향상시킬 수 있습니다.
 
-JavaScript에서 불변성의 데이터를 다루는 것은 [Clojure](http://clojure.org/)같이 그것을 위해 디자인된 언어로 다루는 것보다는 어렵습니다. 하지만, React는 간단한 불변성 헬퍼를 제공합니다. `update()`는 이런 종류의 데이터를 근본적인 변화 *없이*  쉽게 다루도록 해줍니다.
+JavaScript에서 불변성의 데이터를 다루는 것은 [Clojure](http://clojure.org/)같이 그것을 위해 디자인된 언어로 다루는 것보다는 어렵습니다. 하지만, React는 간단한 불변성 헬퍼를 제공합니다. `update()`는 이런 종류의 데이터를 근본적인 변화 *없이*  쉽게 다루도록 해줍니다. Immutable-js에 관한 좀 더 자세한 정보는 페이스북의 [Immutable-js](https://facebook.github.io/immutable-js/docs/)나 [고급 성능](react/docs/advanced-performance-ko-KR.html)을 참조하세요.
```
Comment (arkist) [inline docs/docs/10.7-update.ko-KR.md]: "성능 심화"로 되어있습니다

Comment (marocchino) [inline docs/docs/10.7-update.ko-KR.md]: Applied. :3 6f434b3

**Comment by arkist** [comment]
:+1:

**Comment by zpao** [comment]
Thanks again! You're all awesome :)

**Pull request merged**